onnx/optimizer/passes/fuse_bn_into_conv.h

```diff
@@ -0,0 +1,206 @@
+
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+#include <math.h>
+
+namespace ONNX_NAMESPACE { namespace optimization {
+//TODO: Currently broken for complex values and float16
+struct FuseBNIntoConv final : public OptimizePass {
+  explicit FuseBNIntoConv()
+    : OptimizePass("fuse_bn_into_conv", API_TYPE::IR) {
+  }
+
+  template<typename T>
+  static void add_nums(void* x, const void* y) {
```

Comment:
move these to tensor.h, and define add, sub, mul, div on the Tensor class that applies the corresponding operation to the whole tensor.